riscv/insns/sspush_x1.h

```diff
@@ -0,0 +1,8 @@
+#include "zicfiss.h"
+require_extension(EXT_ZICFISS);
+
+if (xSSE()) {
+  reg_t ssp = STATE.ssp->read();
+  store_to_ssp(ssp - xlen / 8, RS2);
+  STATE.ssp->write(ssp - xlen / 8);
+}
```

Comment:
The small duplication of code between compressed and uncompressed sspush and sspopchk seems unfortunate. Since we now have a zicfiss.h is it possible to have a sspush and a sspopchk macro that can be invoked by the 3 sspush and 3 sspopchk respectively?

Reply:
After refactoring the code, I think I can organize it into `PUSH_VALUE_TO_SS` and `POP_VALUE_FROM_SS_AND_CHECK`.

Thanks for the suggestion